ui/src/app/pages/login/account-creation/account-creation.tsx

```diff
@@ -566,15 +585,41 @@ export class AccountCreation extends React.Component<AccountCreationProps, Accou
               </FlexRow>
             </FlexColumn>
           </Section>
-          <Section sectionHeaderStyles={{borderBottom: null}} header={<React.Fragment>
-            <div>Please describe your research background, experience, and research interests</div>
-            <div style={styles.asideText}>This information will be posted publicly on the <i>All of Us</i> Research Hub website
-              to inform program participants. <span  style={{marginLeft: 2,
-                fontSize: 12}}>(2000 character limit)</span>
-              <i style={{...styles.publiclyDisplayedText, marginLeft: 2}}>
-                Publicly displayed
-              </i></div>
-          </React.Fragment>}>
+          <Section
+            header={<React.Fragment>
+              <FlexRow style={{alignItems: 'flex-start'}}>
+                <div style={{width: '70%'}}>Your research background, experience, and research interests</div>
+                <PubliclyDisplayed style={{marginLeft: '1.2rem'}}/>
+              </FlexRow>
+              <div
+                style={{...styles.asideText, marginTop: '.125px'}}>
+                  This information will be posted publicly on the <i>All of Us</i> Research Hub website to inform program participants.
+                  <span style={{marginLeft: 2, fontSize: 12}}>(2000 character limit)</span>
+              </div>
+              <div style={{marginTop: '.5rem'}}>
+                <FlexRow style={{color: colors.accent, alignItems: 'center'}}>
+                  <div
+                      style={{cursor: 'pointer', fontSize: 14}}
+                      onClick={() => this.setState(
+                        (previousState) => ({showMostInterestedInKnowingBlurb: !previousState.showMostInterestedInKnowingBlurb})
+                      )}
+                  >
+                    <i>All of Us</i> participants are most interested in knowing:
+                  </div>
+                  <ClrIcon shape='angle' style={{
+                    transform: this.state.showMostInterestedInKnowingBlurb ? 'rotate(180deg)' : 'rotate(90deg)'
+                  }}/>
+                </FlexRow>
+                {this.state.showMostInterestedInKnowingBlurb &&
+                  <ul style={{...styles.asideList, listStylePosition: 'outside', marginLeft: '0.75rem'}}>
```

Comment:
can the listStylePositon move inside asideList ?

Reply:
Question as i have not tried this, do we need style asideList both at ul and li level?

Reply:
We don't - we are using `asideText` on the `li`s.